test/rules/_integration/enable-disable/test.ts.lint

```diff
@@ -78,9 +78,6 @@ var AAAaA = 'test' // previously both `quotemark` and `variable-name` rules were
 var AAAaA = 'test' // tslint:disable-line:quotemark
 var AAAaA = 'test' // ensure that disable-line rule correctly handles previous `enable rule - disable all` switches
 
-// tslint:enable:no-var-keyword
```

Comment:
This conflicts with an assertion in `ruleLoaderTests.ts` that it "loads disabled rules if rule in enableDisableRuleMap". (It also now conflicts with `enable-via-comment`.) Which should we choose?